tests/admin/test-class-admin.php

```diff
@@ -1,5 +1,25 @@
 <?php
 
+class WPSEO_Admin_Double extends WPSEO_Admin {
+
+	/**
+	 * Array with all pages of the plugin.
+	 *
+	 * @var array
+	 */
+	private $wpseo_pages = array(
```

Comment:
If you alter the `$wpseo_pages` in the class to be a constant instead of a variable, this test double would become redundant.